package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0-rc.3",
   "description": "An engine for drawing timeline graph.",
   "scripts": {
-    "lint": "eslint 'src/**/*.ts'",
-    "fix-lint": "eslint 'src/**/*.ts' --fix",
-    "docs": "typedoc './src/*.ts' --exclude **/node_modules/** --out docs/ --readme ./index.docs.md --tsconfig ./tsconfig.json --gitRemote upstream --name \"Short Night TypeScript Modules\" && nodetouch docs/.nojekyll",
+    "lint": "eslint \"src/**/*.ts\"",
```

Comment:
What's the difference between `"` and `'`?

Reply:
With no quotes: The blob isn't interpreted correctly at all and misses files.
WIth single quote: Works in Linux but errors in Windows
With double quote: Works in both.